database/migrations/2020_10_01_055547_create_task_changes_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTaskChangesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('task_changes', function (Blueprint $table) {
```

Comment:
what is this?

Reply:
like a history table. Its just a simple changes table. So im going to make an event and when its triggered it will just update this table with a users name a field like "Status","Priority",... and then the old / new values, that way I can display them if we need to on the task

Like: "At 8:04pm Josh closed the task"

Reply:
Nice! Makes more sense now. thanks